components/landing/TopCollectionsItem.vue

```diff
@@ -12,24 +12,40 @@
             rounded
             :src="collection.image || '/placeholder.webp'" />
         </div>
-        <div class="px-2">
+        <div class="px-2 is-flex is-flex-grow-1 is-flex-direction-column">
           <div class="has-text-weight-bold">
             {{ collection.name | truncateStr(12) }}
           </div>
-          <div class="has-text-grey is-size-7">
-            <div v-if="collection.floorPrice">
-              Floor: <CommonTokenMoney :value="collection.floorPrice" inline />
+          <div class="is-flex is-justify-content-space-between nowrap">
+            <div>
+              <div v-if="collection.floorPrice">
+                Floor:
+                <CommonTokenMoney :value="collection.floorPrice" inline />
+              </div>
+              <div v-else>---</div>
+            </div>
+            <div class="is-uppercase has-text-grey pl-2">
+              {{ urlPrefix }}
             </div>
-            <div v-else>---</div>
           </div>
         </div>
       </div>
-      <div class="is-pulled-right has-text-right px-4">
-        <div>
-          <CommonTokenMoney :value="collection.volume" inline />
-        </div>
-        <div class="has-text-grey is-size-7">
-          avg. <CommonTokenMoney :value="collection.averagePrice" inline />
+      <div class="is-pulled-right has-text-right px-2">
+        <div class="is-flex">
+          <div>
+            <div class="nowrap">
+              <CommonTokenMoney :value="volume" inline />
+            </div>
+            <div class="nowrap">
+              <BasicMoney :value="usdValue" inline :unit="'USD'" />
+            </div>
+          </div>
+          <div
+            class="is-flex is-justify-content-center is-align-items-center pl-2 is-size-5">
+            <p :class="color">
+              {{ diffPercentString }}
+            </p>
+          </div>
```

Comment:
font-size; hide div if empty, not a paragraph
```suggestion
          <div
            v-if="diffPercentString"
            class="is-flex is-justify-content-center is-align-items-center pl-2 is-size-6" :class="color">
            {{ diffPercentString }}
          </div>
```